src/milind/Array.java

```diff
@@ -0,0 +1,78 @@
+/*Assignment - 12 : 7th Oct'2020 
+write a single program having different methods.
+ 
+a) To find Min number from given array
+b) To find Max number from given array
+c) To find Avg of all numbers from give array
+d) To display all numbers [0 to length]
+e) To display all numbers [length to 0]
+g) Method to find average of min and max number from array.*/
+
+package milind;
+
+public class Array {
+	
+	int minNumberFromArray(int[] array) {
+		int minNumber = array[0];
+		for(int index = 0; index < array.length; index++) {

```

Comment:
Video sent on skype for this code, pls go through it

Also check this code for reference 
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/318/files